skops/hf_hub/_hf_hub.py

```diff
@@ -103,4 +187,23 @@ def push(*, repo_id: str, source: Union[str, Path], token: str = None):
     This function raises a ``TypeError`` if the contents of the source folder
     do not make a valid HuggingFace Hub scikit-learn based repo.
     """
-    pass
+    _validate_folder(path=source)
+    client = HfApi()
+
+    if create_remote:
+        try:
+            client.model_info(repo_id=repo_id, token=token)
+        except HTTPError:
+            client.create_repo(repo_id=repo_id, token=token, repo_type="model")
+
+    client.upload_folder(
+        repo_id=repo_id,
+        path_in_repo=".",
+        folder_path=source,
+        commit_message=commit_message,
+        commit_description=None,
+        token=token,
+        repo_type=None,
```

Comment:
This is by default None and None refers to model BTW

Reply:
Yes, same as above, I'm just explicitly setting it in case in the future the default value changes.

Reply:
ah gotcha!